File: Plov/src/Knife.java

public class Knife {

	public void Cut_onion(Onion o)
    {
	  if (o.getCut_ready()==false)
      {
          o.setCut_ready(true);

      }
    }
	
	
	public void Cut_meat(Meat m)
    {
	  if (m.getCut_ready()==false)
      {
          m.setCut_ready(true);

      }
    }
	
	public void Cut_carrot(Carrot c)
    {
	  if (c.getCut_ready()==false)
      {
          c.setCut_ready(true);

      }
    }
	
}

File: Laba1Plov/Laba1Plov/Knife.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laba1Plov
{
    class Knife
    {

        public void Сut_onion(Onion o)
        {
            if (o.Has_Сut) o.Has_Сut = false;

        }

        public void Сut_carrot(Carrot c)
        {
            if (c.Has_Сut) c.Has_Сut = false;

        }

        public void Cut_meat(Meat m)
        {
            if (m.Has_Сut) m.Has_Сut = false;

        }
    }
}
